### PR TITLE
cloud-cost-volume-logic

### PR DESCRIPTION
**Pull request by JesseGoodier**
## What does this PR change?
This gives preference to federated-store over object-store for cloud costs. Should only fix issues and be backwards compatible.

### CODE REVIEW

**Comment by jessegoodier** [comment]
Not going to merge yet, but want to get your thoughts @Sean-Holcomb. Does this make sense to you? 
I found an environment where cloud-cost was looking for `object-store.yaml` but the file was `federated-store.yaml`

**Comment by jessegoodier** [comment]
Not going to merge yet, but want to get your thoughts @Sean-Holcomb. Does this make sense to you? 
I found an envrironment where cloud-cost was looking for `object-store.yaml` but the file was `federated-store.yaml`

File: cost-analyzer/templates/_helpers.tpl [hunk 1]
```diff
@@ -1040,8 +1040,7 @@ Begin Kubecost 2.0 templates
     - name: federated-storage-config
       mountPath: /var/configs/etl/federated
       readOnly: true
-  {{- end }}
-  {{- if .Values.kubecostModel.etlBucketConfigSecret }}
+  {{- else }}
```
Comment (Sean-Holcomb) [inline cost-analyzer/templates/_helpers.tpl]: I am not sure about this. when running as a container, `cloud-cost` doesn't depend on a secret

Comment (Sean-Holcomb) [reply cost-analyzer/templates/_helpers.tpl]: maybe an else if

**Comment by Sean-Holcomb** [comment]
Proposal for a clean up effort, making it so there is only one bucket configuration

**Pull request closed without merging**